lib/preprocessors/glimmer.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+const { getTemplateLocals } = require('@glimmer/syntax');
+const {
+  preprocessEmbeddedTemplates,
+} = require('ember-template-imports/lib/preprocess-embedded-templates');
+const util = require('ember-template-imports/src/util');
+
+const TRANSFORM_CACHE = new Map();
+const TEXT_CACHE = new Map();
+
+function gjs(text, filename) {
```

Comment:
Can you add a high-level comment to this function explaining what it does?

Reply:
done